app/views/questions/health_insurance/edit.html.erb

```diff
@@ -1,3 +1,25 @@
-<% content_for :form_question, t("views.questions.health_insurance.title", :year => current_intake.most_recent_filing_year, :count => current_intake.filer_count) %>
+<% content_for :page_title, t('views.questions.health_insurance.title.one', :year => current_intake.most_recent_filing_year, :count => current_intake.filer_count) %>
 
-<% content_for :form_help_text, t("views.questions.health_insurance.help_text") %>
\ No newline at end of file
+<% content_for :card do %>
+  <%= form_with model: @form, url: current_path, local: true, method: "put", builder: VitaMinFormBuilder, html: { class: "form-card" } do |f| %>
+    <h1 class="form-question spacing-below-35">
+      <%= content_for(:page_title) %>
+    </h1>
+    <p>
+      <%=t('views.questions.work_situations.help_text') %>
+    </p>
+    <div class="form-card__stacked-checkboxes spacing-above-0">
+      <%= f.cfa_checkbox(:bought_health_insurance, t('views.questions.health_insurance.options.marketplace_hi_html', :count => current_intake.filer_count), options: { checked_value: "yes", unchecked_value: "no" }) %>
```

Comment:
remove count maybe? confirm whether copy should be different for single vs other filers

Reply:
actually this would be useful for conjugating spanish verbs